pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -86,20 +86,25 @@ func (c *cyclonedxParser) getTopLevelPackage(cdxBom *cdx.BOM) error {
 					repositoryURL = splitImage[0] + "/" + splitImage[1] + "/" + splitTag[0]
 				case 2:
 					repositoryURL = splitImage[0] + "/" + splitTag[0]
+				case 1:
+					repositoryURL = splitImage[0]
 				default:
 					repositoryURL = ""
 				}
 
 				if len(splitTag) == 2 {
 					tag = splitTag[1]
 				}
-
-				purl = "pkg:guac/cdx/" + repositoryURL + "@" + cdxBom.Metadata.Component.Version + "?tag=" + tag
+				if repositoryURL != "" {
+					purl = "pkg:guac/cdx/" + repositoryURL + "@" + cdxBom.Metadata.Component.Version + "?tag=" + tag
```

Comment:
`cdxBom.Metadata.Component.Version` is not required. Will have to check for this.